content/posts/2025/03/lesson-setup-instructions-task-force-recommendations.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: page
+authors: ["Toby Hodges"]
+teaser: "Summarising the recommendations of the Lesson Setup Instructions Task Force"
+title: "Updating our Lesson Setup Instructions"
+Date: 2025-03-12
+time: "09:00:00"
+tags: ["Curriculum", "Python", "R", "Shell"]
+---
+
+The Lesson Setup Instructions Task Force was [established in August 2024 to review existing setup instructions](https://github.com/carpentries/task-forces/blob/main/2024/lesson-setup/lesson-setup-instructions.md) for Data Carpentry, Library Carpentry, and Software Carpentry lessons and decide where and how these instructions should be updated. Members of the task force considered a wide range of options for installing Bash, Git, Python, and R, and engaged in many constructive discussions. The task was challenging, and the “best” option was often difficult to identify.
+
+**Task force members:** Toby Hodges (Chair), Rob Davey, Matt Fisher, Nathaniel Porter, Jennifer Stubbs, Adam Taranto
+
+The full text of the task force’s recommendations can be found in the `carpentries/task-forces` repository on GitHub. Here we provide a summary, highlighting the changes that we expect to impact Instructors the most.
```

Comment:
```suggestion
[The full text of the task force’s recommendations](https://github.com/carpentries/task-forces/blob/main/2024/lesson-setup/lsitf-recommendations.md) can be found in the `carpentries/task-forces` repository on GitHub. Here we provide a summary, highlighting the changes that we expect to impact Instructors the most.
```

This URL will work after https://github.com/carpentries/task-forces/pull/41 has been merged.